### PR TITLE
EDURED-54: The input doesn't follow regex

### PR DESCRIPTION
**Pull request by risinggalaxy**

### CODE REVIEW

File: Sources/EduIDExpansion/Utilities/Constants.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//  Created by Yasser Farahi on 06/04/2023.
+//
+
+import Foundation
+
+enum Constants {
+    
+    enum RegEx {
+        static let emailRegex = #"^(?=.{6,})[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$"#
+        static let nameRegex = #"^[a-zA-Z' ]+$"#
```
Comment (johankool) [inline Sources/EduIDExpansion/Utilities/Constants.swift]: Perhaps a bit too strict still? For example a dash can happen too: Willem-Alexander.

File: Sources/EduIDExpansion/Utilities/Constants.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        static let passwordRegex = #"(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?]).{8,}"#
+        static let phoneRegex = #"^(?=[+0])[+\d\s]*(?:\d\s*){10,}$"#
```
Comment (johankool) [inline Sources/EduIDExpansion/Utilities/Constants.swift]: If read this regex it seems to only allow for ten digit numbers. That's true in the Netherlands but not around the world.

**Review by johankool — APPROVED** [review]
Looks good, but perhaps regex needs more tweaking.

File: Sources/EduIDExpansion/Utilities/Constants.swift (hunk 2 above)
Comment (risinggalaxy) [reply Sources/EduIDExpansion/Utilities/Constants.swift]: @johankool well the phone number can start wit a + or a single zero, can contain whitespaces(no dashes the US & Canada ways are not supported you are right)and should be at least ten digits, what do you suggest?

File: Sources/EduIDExpansion/Utilities/Constants.swift (hunk 1 above)
Comment (risinggalaxy) [reply Sources/EduIDExpansion/Utilities/Constants.swift]: @johankool you are 100% right, I'll make the adjustments.

**Pull request merged**